crates/bevy_app/src/app.rs

```diff
@@ -76,22 +76,27 @@ struct SubApp {
     runner: Box<dyn Fn(&mut World, &mut App)>,
 }
 
-impl Default for App {
-    fn default() -> Self {
-        let mut app = App::empty();
-        #[cfg(feature = "bevy_reflect")]
-        app.init_resource::<AppTypeRegistry>();
-
-        app.add_default_stages()
-            .add_event::<AppExit>()
-            .add_system_to_stage(CoreStage::Last, World::clear_trackers.exclusive_system());
+/// Sets the default stages to be single or multi threaded.
+pub enum AppThreading {
+    /// run app single threaded
+    Single,
+    /// run app multi threaded
+    Multi,
+}
 
-        #[cfg(feature = "bevy_ci_testing")]
-        {
-            crate::ci_testing::setup_app(&mut app);
+impl AppThreading {
+    /// Gets a new instance of [`SystemStage`] for the corresponding value of [`AppThreading`]
+    pub fn get_stage(&self) -> SystemStage {
+        match self {
+            AppThreading::Multi => SystemStage::parallel(),
+            AppThreading::Single => SystemStage::single_threaded(),
         }
+    }
+}
 
-        app
+impl Default for App {
+    fn default() -> Self {
+        App::single_threaded()
```

Comment:
```suggestion
        App:: multi_threaded()
```
Shouldn't the default still be multi threaded?